internal/hardware/virt/virt.go

```diff
@@ -11,5 +12,5 @@ const (
 )
 
 func IsVirtSupported(inventory *models.Inventory) bool {
-	return funk.Contains(inventory.CPU.Flags, intelVirtCpuFlag) || funk.Contains(inventory.CPU.Flags, amdVirtCpuFlag)
+	return slices.Contains(inventory.CPU.Flags, intelVirtCpuFlag) || slices.Contains(inventory.CPU.Flags, amdVirtCpuFlag) || inventory.CPU.Architecture == models.InfraEnvCPUArchitectureArm64
```

Comment:
Can you please add a TODO comment telling that we need to find out how to whether ARM cpu host is virt supported?